crates/holochain/src/core/workflow/sys_validation_workflow.rs

```diff
@@ -247,42 +552,66 @@ fn handle_failed(error: &ValidationOutcome) -> Outcome {
     }
 }
 
+fn action_to_entry_rate_weight(action: &Action) -> SysValidationResult<EntryRateWeight> {
```

Comment:
Extracted from existing code so I can share them between dependency fetching and validation